Core/AppRuntime/Source/TaskChain.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include "Common.h"
+
+#include <arcana/threading/task.h>
+#include <napi/env.h>
+
+namespace Babylon
+{
+    class TaskChain
```

Comment:
Consider naming this `WorkQueue` (or `JSWorkQueue`) (I feel like that is a more standard name for this concept).